CRM/CivirulesActions/Participant/AddToZoom.php

```diff
@@ -96,17 +97,18 @@ private function addParticipant($participant, $webinar) {
 			$lastName = $participant['last_name'];
 
 			CRM_Core_Session::setStatus(
-				"$firstName $lastName was added to Zoom Webinar $webinar.", 
-				ts('Participant added!'), 
+				"$firstName $lastName was added to Zoom Webinar $webinar.",
+				ts('Participant added!'),
 				'success'
 			);
 		}
 	}
 
 	private function createJWTToken() {
-		$key = $_ENV['ZOOM_API_SECRET'];
+		$settings = CRM_NcnCiviZoom_Utils::getZoomSettings();
+		$key = $settings['api_key'];
```

Comment:
looks like you have used the wrong key names here. you have misplaced the api_key and secret key in wrong variables. $key should be Secret Key and $payload['iss'] should be API key.